main.c

```diff
@@ -36,97 +36,77 @@ struct {
 
 void main_initthr(void *unused)
 {
-	size_t i;
-	syspage_program_t prog;
-	int xcount = 0, res;
-	char *cmdline = syspage->arg, *end;
-	char *argv[32], *arg, *argend;
+	int res;
+	unsigned int argc;
+
+	syspage_prog_t *prog;
+	char *argv[32], *cmdline;
 
 	/* Enable locking and multithreading related mechanisms */
 	_hal_start();
 
-	lib_printf("main: Decoding programs from data segment\n");
-	programs_decode(&main_common.kmap, &main_common.kernel);
-
 	lib_printf("main: Starting syspage programs:");
-	for (i = 0; i < syspage->progssz; i++)
-		lib_printf(" '%s',", syspage->progs[i].cmdline);
-	lib_printf("\b \n");
+	syspage_progShow();
 
 	posix_init();
 	posix_clone(-1);
 
+
 	/* Free memory used by initial stack */
 	/*vm_munmap(&main_common.kmap, main_common.stack, main_common.stacksz);
 	vm_pageFree(p);*/
 
 	/* Set stdin, stdout, stderr ports */
-//	proc_fileAdd(&h, &oid, 0);
-//	proc_fileAdd(&h, &oid, 0);
-//	proc_fileAdd(&h, &oid, 0);
+	/*
+	proc_fileAdd(&h, &oid, 0);
+	proc_fileAdd(&h, &oid, 0);
+	proc_fileAdd(&h, &oid, 0);
+	*/
```

Comment:
`//` was used here to indicate that the code has been disabled temporally.
It's disabled for quite some time, let's remove it for now (we have version control to store old code).

Reply:
I have changed  `//` to `/* */` to be consistent with our coding standard. 
I was willing to remove it before, but I wasn't sure whether I would be still used somewhere.
 
What do you think about removing this part of code as well?
https://github.com/phoenix-rtos/phoenix-rtos-kernel/blob/8538c0c28fd74847fa383e5b6f665660b0bcb882/main.c#L134-L141

Reply:
`/* */` should be used for comments, that's true, but this is not a comment - it's a disabled code.
`#if 0` would be most right way to do it. But I think this code is dead for now, no need to keep it.
Don't remove tests for now, they are being used sometimes.

Reply:
Done